spec.md

```diff
@@ -431,7 +431,7 @@ Blob properties:
 
 ### Resource Data Model
 
-A resource is a named (addressable) object stored in a Space, with metadata.
+A resource is a named (addressable) object stored in a [=space=], with metadata.
```

Comment:
what is this syntax?

Reply:
ReSpec's "link to the definition of the term" syntax.

Reply:
ty